src/pages/index.jsx

```diff
@@ -1,35 +1,23 @@
 import dynamic from 'next/dynamic';
-import {useRouter} from 'next/router';
+import Head from 'next/head';
+
+import Layout from '../components/Layout';
 
-import StyledButton from '../components/styledButton';
 export default function Homepage() {
-	const router = useRouter();
 	const DynamicWrapper = dynamic(
 		() => import('../components/Forms/StyledComponents/styledFormWrapper'),
 		{
 			ssr: false,
 		}
 	);
 	return (
-		<DynamicWrapper>
-			<StyledButton
-				onClick={() => {
-					router.push({
-						pathname: '/clients',
-					});
-				}}
-			>
-				Clients
-			</StyledButton>
-			<StyledButton
-				onClick={() => {
-					router.push({
-						pathname: '/services',
-					});
-				}}
-			>
-				Services
-			</StyledButton>
-		</DynamicWrapper>
+		<Layout>
+			<Head>
+				<title key="title">Dashy</title>
+				<meta key="description" name="description" content="This is my Capstone project" />
+				<link rel="icon" href="/Dashy.webp" />
+			</Head>
+			<DynamicWrapper></DynamicWrapper>
```

Comment:
You don't need the closing tag
```suggestion
			<DynamicWrapper />
```